src/next-steps/next-steps.module.css

```diff
@@ -194,4 +194,13 @@ input[type='checkbox'].taskCheckbox:checked::after {
 		margin-top: 3rem;
 		margin-left: 0;
 	}
+
+	.taskListItem {
+		flex-wrap: wrap;
+	}
+
+	.buttonContent {
+		margin-top: 0.625rem;
```

Comment:
This is awesome, Karen! Thank you for making the change 👍 

**nitpick (non-blocking):** This is a super minor aesthetic nitpick, but would it be possible to increase `margin-top` slightly? Perhaps to `1rem`? It feels a bit close to the description text and could use a little more separation. I know you only had my grainy screenshot to go off of 😆, so apologies for not providing more guidance!

Reply:
You're good! I've updated the `margin-top` to `1rem` in [0d0e413](https://github.com/Automattic/bugomattic/pull/73/commits/0d0e4138ed5e09cc9ee81cf1091941dbba318386).